R/serialization.R

```diff
@@ -0,0 +1,42 @@
+# This file will be autogenerated.
+
+#' Serialize/Unserialize an mlpack model.
+#'
+#' @title Mlpack-Serialization.
+#' @param filename Input filename.
+#' @param model Input model pointer.
+#' @export
+#' @rdname mlpack-serialization
+Serialize <- function(filename, model) {
```

Comment:
@Yashwants19 minor nit:

Can we swap positions of `filename` and `model` parameters?

```suggestion
Serialize <- function(model, filename) {
```

This will -- sorta -- match with `serialize()` parameter arrangement. 

```r
serialize(object, connection)
```

Reply:
Done. :)